js/index.d.ts

```diff
@@ -0,0 +1,54 @@
+// Type definitions for apptentive-react-native
+// Project: apptentive-react-native
+// Definitions by: Matthew Callis
+
+export = ApptentiveConfiguration;
```

Comment:
Authoring TS declarations is still a bit of a black art, but reassigning `export` to `Apptentive` would then "hide" `ApptentiveConfiguration`? At least if it obeys the laws of JavaScript. There is probably multiple syntaxes that work, but I think you could have a single `export {ApptentiveConfiguration, Apptentive}`. You might need to try importing with a dummy `.ts` file to verify.